venia-integration-tests/src/fixtures/pageBuilder/block/cmsBlock.json

```diff
@@ -0,0 +1,20 @@
agento&quot;,&quot;position&quot;:{&quot;latitude&quot;:30.267199999999995,&quot;longitude&quot;:-97.73760683593753},&quot;phone&quot;:&quot;(512) 123-4567&quot;,&quot;address&quot;:&quot;1234 Here Ln&quot;,&quot;city&quot;:&quot;Austin&quot;,&quot;state&quot;:&quot;TX&quot;,&quot;zipcode&quot;:&quot;77840&quot;,&quot;country&quot;:&quot;United States&quot;,&quot;comment&quot;:&quot;Line 1\\nLine 2&quot;,&quot;record_id&quot;:0}]\" data-element=\"main\" style=\"border-style: none; border-width: 1px; border-radius: 0px; margin: 0px; padding: 0px; height: 300px;\"></div></div></div></div><div data-content-type=\"row\" data-appearance=\"full-bleed\" data-enable-parallax=\"1\" data-parallax-speed=\"0.5\" data-background-images=\"{\\&quot;desktop_image\\&quot;:\\&quot;{{media url=wysiwyg/home/home-main.jpg}}\\&quot;,\\&quot;mobile_image\\&quot;:\\&quot;{{media url=wysiwyg/home/home-erin.jpg}}\\&quot;}\" data-element=\"main\" style=\"justify-content: center; display: flex; flex-direction: column; background-color: rgb(168, 208, 152); background-position: right center; background-size: contain; background-repeat: repeat; background-attachment: scroll; border-style: none; border-width: 1px; border-radius: 0px; min-height: 700px; margin: 0px 0px 10px; padding: 10px;\"><h1 data-content-type=\"heading\" data-appearance=\"default\" data-element=\"main\" style=\"text-align: center; border-style: none; border-width: 1px; border-radius: 0px;\">Block</h1><div data-content-type=\"products\" data-appearance=\"grid\" data-element=\"main\" style=\"text-align: center; border-style: none; border-width: 1px; border-radius: 0px; margin: 0px; padding: 0px;\">{{widget type=\"Magento\\CatalogWidget\\Block\\Product\\ProductsList\" template=\"Magento_CatalogWidget::product/widget/content/grid.phtml\" anchor_text=\"\" id_path=\"\" show_pager=\"0\" products_count=\"2\" condition_option=\"condition\" condition_option_value=\"\" type_name=\"Catalog Products List\" conditions_encoded=\"^[`1`:^[`type`:`Magento||CatalogWidget||Model||Rule||Condition||Combine`,`aggregator`:`all`,`value`:`1`,`new_child`:``^],`1--1`:^[`type`:`Magento||CatalogWidget||Model||Rule||Condition||Product`,`attribute`:`category_ids`,`operator`:`==`,`value`:`3`^]^]\" sort_order=\"name_descending\"}}</div></div><div data-content-type=\"row\" data-appearance=\"contained\" data-element=\"main\" style=\"display: none;\"><div data-enable-parallax=\"0\" data-parallax-speed=\"0.5\" data-background-images=\"{}\" data-element=\"inner\" style=\"justify-content: flex-start; display: flex; flex-direction: column; background-position: left top; background-size: cover; background-repeat: no-repeat; background-attachment: scroll; border-style: none; border-width: 1px; border-radius: 0px; margin: 0px 0px 10px; padding: 10px;\"><h2 data-content-type=\"heading\" data-appearance=\"default\" data-element=\"main\" style=\"text-align: center; border-style: none; border-width: 1px; border-radius: 0px;\">Hidden Row</h2></div></div>",
```

Comment:
The `block.json` looks updated, but the remaining three fixtures appear to have old PB content data with inline styles. Do these three fixtures need updated?

Reply:
The fixtures have been updated.